<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Medienverwaltung</class>
 <widget class="QMainWindow" name="Medienverwaltung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medienverwaltung</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tab_view">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_media">
       <attribute name="title">
        <string>Medien</string>
       </attribute>
       <widget class="QLabel" name="lbl_mediaHeader">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>801</width>
          <height>47</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Medien</string>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
       <widget class="QGroupBox" name="gB_addMedia">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>921</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Medium hinzufügen</string>
        </property>
        <widget class="QComboBox" name="cB_addMedia_type">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>25</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_addMedia_title">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>341</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Titel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_addMedia">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>20</y>
           <width>211</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Medium hinzufügen</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_addMedia_creator">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>30</y>
           <width>251</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Autor/ Interpret/ Regisseur</string>
         </property>
        </widget>
       </widget>
       <widget class="QTableWidget" name="db_media">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>931</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="gB_Suche">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>931</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Suche</string>
        </property>
        <widget class="QComboBox" name="cB_searchMedia_type">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>86</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_searchMedia_title">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>341</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Titel</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_searchMedia_creator">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>30</y>
           <width>241</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Autor/ Interpret/ Regisseur</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_searchMedia">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>20</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Suchen</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_resetSearchMedia">
         <property name="geometry">
          <rect>
           <x>820</x>
           <y>20</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Zurücksetzen</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="gB_lendMedia_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>510</y>
          <width>921</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Medium ausleihen</string>
        </property>
        <widget class="QPushButton" name="btn_lendMedia">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>20</y>
           <width>211</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Ausleihen</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cB_lendMedia_media">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>30</y>
           <width>341</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="cB_lendMedia_user">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>341</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="gB_returnMedia">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>590</y>
          <width>921</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Medium zurückgeben</string>
        </property>
        <widget class="QPushButton" name="btn_returnMedia">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>20</y>
           <width>211</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Zurückgeben</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cB_returnMedia_media">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>30</y>
           <width>341</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="cB_returnMedia_user">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>341</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
       </widget>
       <zorder>gB_addMedia</zorder>
       <zorder>lbl_mediaHeader</zorder>
       <zorder>db_media</zorder>
       <zorder>gB_Suche</zorder>
       <zorder>gB_lendMedia_2</zorder>
       <zorder>gB_returnMedia</zorder>
      </widget>
      <widget class="QWidget" name="tab_users">
       <attribute name="title">
        <string>Personen</string>
       </attribute>
       <widget class="QLabel" name="lbl_userHeader">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1006</width>
          <height>47</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Registrierte Nutzer</string>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
       <widget class="QTableWidget" name="db_user">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>931</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="gB_registerUser">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>420</y>
          <width>371</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>Nutzer registrieren</string>
        </property>
        <widget class="QLineEdit" name="lE_registerUser_name">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>191</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_name">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Vorname:</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_surname">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Nachname:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_registerUser_surname">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>80</y>
           <width>191</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_registerUser">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>311</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Registrieren</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_birthdate">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Geburtsdatum:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_registerUser_email">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>191</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_email">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dE_registerUser_birthdate">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>140</y>
           <width>191</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="gB_lentMedia">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>420</y>
          <width>411</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>ausgeliehene Medien</string>
        </property>
        <widget class="QListWidget" name="li_lentMedia">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>30</y>
           <width>381</width>
           <height>141</height>
          </rect>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_returnMedia_user">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>180</y>
           <width>311</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Zurückgeben</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="gB_searchUser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>931</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Suche</string>
        </property>
        <widget class="QLineEdit" name="lE_searchUser_name">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>191</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Vorname</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_searchUser_surname">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>30</y>
           <width>201</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_searchUser">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>20</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Suchen</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_resetSearchUser">
         <property name="geometry">
          <rect>
           <x>820</x>
           <y>20</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Zurücksetzen</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lE_searchUser_email">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>30</y>
           <width>281</width>
           <height>25</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Email</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/img/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save data to files (CTRL+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/import.png</normaloff>:/img/import.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load data from files (CTRL+L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tab_view</tabstop>
  <tabstop>db_media</tabstop>
  <tabstop>cB_addMedia_type</tabstop>
  <tabstop>lE_addMedia_title</tabstop>
  <tabstop>lE_addMedia_creator</tabstop>
  <tabstop>btn_addMedia</tabstop>
  <tabstop>cB_lendMedia_user</tabstop>
  <tabstop>cB_lendMedia_media</tabstop>
  <tabstop>btn_lendMedia</tabstop>
  <tabstop>cB_returnMedia_user</tabstop>
  <tabstop>cB_returnMedia_media</tabstop>
  <tabstop>btn_returnMedia</tabstop>
  <tabstop>cB_searchMedia_type</tabstop>
  <tabstop>lE_searchMedia_title</tabstop>
  <tabstop>lE_searchMedia_creator</tabstop>
  <tabstop>btn_searchMedia</tabstop>
  <tabstop>btn_resetSearchMedia</tabstop>
  <tabstop>db_user</tabstop>
  <tabstop>lE_registerUser_name</tabstop>
  <tabstop>lE_registerUser_surname</tabstop>
  <tabstop>lE_registerUser_email</tabstop>
  <tabstop>dE_registerUser_birthdate</tabstop>
  <tabstop>btn_registerUser</tabstop>
  <tabstop>li_lentMedia</tabstop>
  <tabstop>btn_returnMedia_user</tabstop>
  <tabstop>lE_searchUser_name</tabstop>
  <tabstop>lE_searchUser_surname</tabstop>
  <tabstop>lE_searchUser_email</tabstop>
  <tabstop>btn_searchUser</tabstop>
  <tabstop>btn_resetSearchUser</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
